### PR TITLE
Add package r-lmvar

### PR DESCRIPTION
**Pull request by comatrion**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by comatrion** [comment]
Used https://github.com/bgruening/conda_r_skeleton_helper

**Comment by comatrion** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes, ready for review

File: recipes/r-lmvar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+about:
+  home: https://CRAN.R-project.org/package=lmvar
+  license: GPL-3
+  summary: 'Runs a linear-like regression with in which both the expected value and the variance
+    can vary per observation. The expected values mu follows the standard linear model
+    mu = X_mu * beta_mu. The standard deviation sigma follows the model log(sigma) =
+    X_sigma * beta_sigma. The package comes with two vignettes: ''Intro'' gives an introduction,
+    ''Math'' gives mathematical details.'
+  license_family: GPL3
```
Comment (cbrueffer) [inline recipes/r-lmvar/meta.yaml:51]:
```suggestion
  license_family: GPL-3.0-only
```

**Comment by cbrueffer** [comment]
Looks good otherwise apart from the license fix.

**Comment by mcouch-synthace** [comment]
Ach sorry I spotted that already it's in the second commit https://github.com/conda-forge/staged-recipes/pull/18550/commits/a9d53a5eb3e4ab339257a22203cf0f3f8dd612ee
Should I squash or is it OK?

**Comment by comatrion** [comment]
@cbrueffer - thanks, I believe correct now.

**Pull request merged**